packages/@ember/-internals/glimmer/tests/integration/components/input-angle-test.js

```diff
@@ -316,10 +327,16 @@ moduleFor(
       });
     }
 
-    ['@test sends an action with `<Input @escape-press={{action "foo"}} />` when <escape> is pressed'](
+    [`${testUnless(
+      DEPRECATIONS.DEPRECATE_TEMPLATE_ACTION.isRemoved
+    )} sends an action with \`<Input @escape-press={{action "foo"}} />\` when <escape> is pressed`](
       assert
     ) {
-      assert.expect(2);
+      assert.expect(3);
+      expectDeprecation(
+        /Usage of the `\(action\)` helper is deprecated./,
+        DEPRECATIONS.DEPRECATE_TEMPLATE_ACTION.isEnabled
+      );
```

Comment:
Should this test be converted to using functions?

Reply:
fair enough, updated!